console-src/src/layout/Layout.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment, ReactNode } from "react";
+import styled from "styled-components";
+
+import { ReactComponent as Logo } from "../assets/svg/logo.svg";
+
+const HeaderContainer = styled.header`
+  display: flex;
+  flex-shrink: 0;
+  justify-content: center;
+  padding: 1.5rem;
+  margin-bottom: 1rem;
+`;
+
+const Header = () => (
+  <HeaderContainer>
+    <Logo />
+  </HeaderContainer>
+);
+
+const Footer = styled.footer`
+  height: 30px;
+  width: 100%;
+  flex-shrink: 0;
+`;
+
+const Content = styled.main`
+  width: 660px;
+`;
```

Comment:
Should it be responsive and be suitable for devices with width less than 660px?

Reply:
Yes, it should 🙂 I will update it